### PR TITLE
Update README.md to include MariaDB

### PR DESCRIPTION
**Pull request by robertsilen**
Considering the technical requirements (https://downloads.joomla.org/technical-requirements) mention MariaDB, I humbly suggest the README should include MariaDB too, i.e. MySQL/MariaDB.

Pull Request for Issue # .

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by HLeithner** [comment]
We no longer support, SQL Server, and we support MariaDB not official in 4.x. Started with Joomla 5.0. 
Please remove mariadb und SQL Server from this PR and create a new one for 5.1 branch.

thanks

**Comment by Quy** [comment]
This PR can be closed.

> Please remove mariadb und SQL Server from this PR and create a new one for 5.1 branch.

**Comment by robertsilen** [comment]
> Revert and update readme.txt.
> 
> > Please remove mariadb und SQL Server from this PR and create a new one for 5.1 branch.

I have now modified both READM.txt and README.md in this PR for Joomla 4.4 to include MariaDB

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Overview
 What is Joomla?
 ---------------------
 * [Joomla!](https://www.joomla.org/about-joomla.html) is a **Content Management System** (CMS) which enables you to build websites and powerful online applications.
-* It is a simple and powerful web server application which requires a server with PHP and either MySQL or PostgreSQL to run. You can find [full technical requirements here](https://downloads.joomla.org/technical-requirements).
+* It is a simple and powerful web server application which requires a server with PHP and either MySQL, MariaDB or PostgreSQL to run. You can find [full technical requirements here](https://downloads.joomla.org/technical-requirements).
```
Comment (Quy) [inline README.md:20]:
```suggestion
* It is a simple and powerful web server application which requires a server with PHP and either MySQL, MariaDB, PostgreSQL or SQL Server to run. You can find [full technical requirements here](https://downloads.joomla.org/technical-requirements).
```
To be consistent with README.txt.

**Comment by Quy** [comment]
Please do as requested per @HLeithner in this PR.

> We no longer support, SQL Server, and we support MariaDB not official in 4.x. Started with Joomla 5.0. Please remove mariadb und SQL Server from this PR and create a new one for 5.1 branch.

File: README.md (hunk 1 above)
Comment (Quy) [inline README.md]:
```suggestion
* It is a simple and powerful web server application which requires a server with PHP and either MySQL or PostgreSQL to run. You can find [full technical requirements here](https://downloads.joomla.org/technical-requirements).
```

File: README.txt [hunk 2]
```diff
@@ -9,7 +9,7 @@ Joomla! CMS™
 2- What is Joomla?
 	* Joomla! is a Content Management System (CMS) which enables you to build websites and powerful online applications.
 	* It's a free and Open Source software, distributed under the GNU General Public License version 2 or later.
-	* This is a simple and powerful web server application and it requires a server with PHP and either MySQL, PostgreSQL or SQL Server to run.
+	* This is a simple and powerful web server application and it requires a server with PHP and either MySQL, MariaDB, PostgreSQL or SQL Server to run.
```
Comment (Quy) [inline README.txt]:
```suggestion
	* This is a simple and powerful web server application and it requires a server with PHP and either MySQL or PostgreSQL to run.
```

**Comment by Quy** [comment]
Go to https://manual.joomla.org/docs/next/get-started/technical-requirements/#requirements-for-joomla-4x to confirm only MySQL and PostgreSQL in v4.

**Comment by HLeithner** [comment]
thx

**Pull request merged**